draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2229,7 +2254,7 @@ uCDN-A                         dCDN-B                         dCDN-C
 
               <t>Description: A regular expression for URI matching. </t>
 
-              <t>Type: A regular expression to match against the URI, i.e against the path-absolute
+              <t>Type: A regular expression to match against the URI, i.e., against the path-absolute
                  and the query string parameters <xref target="RFC3986"/>. The regular expression
                  string MUST be compatible with <xref target="PCRE841">PCRE</xref>.</t>
```

Comment:
Kevin provided this feedback (and I agree with his stance here):

> for URI signing, there was a POSIX vs PCRE debate, and POSIX won.  should the same apply here?  see: https://mailarchive.ietf.org/arch/msg/cdni/_0Y6yyc4IjIY4cB5vunCbSA2ar0/ and https://datatracker.ietf.org/doc/html/rfc9246#name-uri-regular-expression-cont

Should we change this to POSIX regular expressions?

Reply:
I think we should

Reply:
No problem; I will probably make this change in its own commit since it is a bit more significant.

Reply:
Resolved in the most recent commit